### DISCUSSION TITLE
How to change cursor color in virtual mode ?

### DISCUSSION THREAD

**Discussion by Adnios** [vscode-neovim/vscode-neovim#2339] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/1ced5f7a-d8ab-4172-beec-9b687d2fe335)

![image](https://github.com/user-attachments/assets/e399e6d0-2e08-4f9a-9eb0-f0386e56905f)
After setting the cursor color to be red. But in virtual mode, the cursor color is still white. I want to change the cursor color as same as normal mode. This might be vscode issue, but I cannot find the answer.
Thanks in advance.

**Comment by theol0403** [comment, marked as answer]
It is controlled by the neovim `:hi Cursor`

**Comment by Adnios** [comment]
And for people might interest. Another way is 
```
    "workbench.colorCustomizations": {
        "editorCursor.background": "#151515",
        "editorCursor.foreground": "#ff0000",
    },

    "vscode-neovim.highlightGroups.highlights": {

        "Cursor" : {
            "backgroundColor": "theme.editorCursor.foreground",
            "color": "theme.editorCursor.background"
        }
    },
```
